plugins/techdocs/src/plugin.ts

```diff
@@ -31,22 +31,22 @@
 
 import { createPlugin, createRouteRef } from '@backstage/core';
 import { TechDocsHome } from './reader/components/TechDocsHome';
-import { Reader } from './reader/components/Reader';
+import { TechDocsPage } from './reader/components/TechDocsPage';
 
 export const rootRouteRef = createRouteRef({
   path: '/docs',
   title: 'TechDocs Landing Page',
 });
 
 export const rootDocsRouteRef = createRouteRef({
-  path: '/docs/:componentId/*',
+  path: '/docs/:kind/:namespace/:name/*',
```

Comment:
😀 Would it be possible to make this overrideable via an apiRef interface? That way we can still use this with our pre-existing docs storage & build setup.